app/controllers/users.controller.js

```diff
@@ -0,0 +1,15 @@
+const db = require('../models/index');
+const logger = require('../logger');
+
+const { databaseError } = require('../errors');
+const { signUp } = require('../mappers/user.mapper');
+
+exports.createUser = (req, res) => {
+  const dataUser = signUp(req.body);
+  db.User.create(dataUser)
```

Comment:
For example, I should have a user service that allows me to export a method to create users, https://github.com/Wolox/tech-guides/blob/master/nodejs/docs/node-standard-and-best-practices.md#22--services